### PR TITLE
Globally Enables Mood

### PR DESCRIPTION
**Pull request by redmoogle**
# Document the changes in your pull request

Its kinda stupid its a toggle. Removes said toggle. Now everyone is a moody ~~bitch~~

# Why this is good

Its unfair for some of the station to be unaffected by things such as mood slowdown or hunger

# Changelog

:cl:  
tweak: Mood is now globally enabled
/:cl:

### CODE REVIEW

**Comment by Bibby0110** [comment]
lol, i doubt this is even going to be considered

it's not unfair if they can just turn it off either

**Comment by ghost** [comment]
Good change

**Comment by VaelophisNyx** [comment]
no more ignoring mood slowdowns
removes powergaming
pog

**Comment by TheRyeGuyWhoWillNowDie** [comment]
mood is bad and you should feel bad for trying to force everyone to have it

**Comment by redmoogle** [comment]
> mood is bad and you should feel bad for trying to force everyone to have it

Hugbox comment

**Comment by TheRyeGuyWhoWillNowDie** [comment]
> > mood is bad and you should feel bad for trying to force everyone to have it
> 
> Hugbox comment

do you have any actual reason for trying to gimp everyone? or just 'Hugbox lol'

**Comment by redmoogle** [comment]
> > > mood is bad and you should feel bad for trying to force everyone to have it
> > 
> > 
> > Hugbox comment
> 
> do you have any actual reason for trying to gimp everyone? or just 'Hugbox lol'

Its much more fair. You're just trying to make the game easier for yourself. Anyways this isnt up to you.

**Comment by Smudged** [comment]
This is a good change, Either everyone should have mood or no one. 

Also, fulfilling your mood requirements didn't give you that much of an advantage.

And Not-fulfilling them didn't give you that big of a slow-down, and the worst slowdown comes from starving which already slows you down anyways.

File: code/modules/antagonists/creep/creep.dm [hunk 1]
```diff
@@ -46,7 +46,7 @@
 	var/mob/living/M = mob_override || owner.current
 	update_obsession_icons_removed(M)
 	var/mob/living/carbon/human/H = M
-	if(H && !H.mood_enabled)
+	if(H)
 		var/datum/component/C = M.GetComponent(/datum/component/mood)
 		if(C) //we cannot be too sure they may have somehow removed it
 			to_chat(owner, span_danger("Your need for mental fitness vanishes alongside the voices, mood has been disabled."))
```
Comment (tattax) [inline code/modules/antagonists/creep/creep.dm:52]:
```suggestion
```

this and removecomponent()

File: code/modules/mob/living/carbon/human/species_types/felinid.dm [hunk 2]
```diff
@@ -167,7 +167,7 @@
 
 /datum/species/human/felinid/spec_life(mob/living/carbon/human/H)
 	. = ..()
-	if((H.client && H.client.prefs.mood_tail_wagging) && !is_wagging_tail() && H.mood_enabled)
+	if((H.client && H.client.prefs.mood_tail_wagging) && !is_wagging_tail())
```
Comment (tattax) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm:170]:
```suggestion
	if((H?.client?.prefs.mood_tail_wagging) && !is_wagging_tail())
```

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm [hunk 3]
```diff
@@ -64,7 +64,7 @@
 
 /datum/species/lizard/spec_life(mob/living/carbon/human/H)
 	. = ..()
-	if((H.client && H.client.prefs.mood_tail_wagging) && !is_wagging_tail() && H.mood_enabled)
+	if((H.client && H.client.prefs.mood_tail_wagging) && !is_wagging_tail())
```
Comment (tattax) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm:67]:
```suggestion
	if((H?.client?.prefs.mood_tail_wagging) && !is_wagging_tail())
```

Comment (redmoogle) [reply code/modules/mob/living/carbon/human/species_types/lizardpeople.dm:67]: Out of scope & not required

**Comment by Marmio64** [comment]
the title isnt very explicit, you should change it so that people know that this isn't just removing a toggle it is also permanently making it so everybody has to use mood. 

I'm neutral on this since while it could give the bartender and chef more reason to make better drinks/food and all that, mood is also currently really boring and it'd be better to make it more interesting FIRST before you force it on everyone.

**Comment by ToasterBiome** [comment]
.

**Comment by athena413** [comment]
bartender role is now gaming

mood good

**Comment by maxion12345** [comment]
what? mood sucks ass. why would I want to micromanage something like that instead of like, actually playing the game? It's cheeseable as hell to I've seen basically every person with mood enabled sprinting to the holodeck to pet the animals before sprinting away any time they so much as get a negative moodlet. It just isn't an interesting game mechanism for most players, And that's why most keep it disabled (because it's not fun)

**Comment by warface1234455** [comment]
finally good change

**Comment by ShadowDeath6** [comment]
It's an opt-in feature. People purposefully turn it on because they want to. If anything, buff mood so that high mood makes you move faster than normal, then the powergamers can turn on mood. 
Saying that an opt-in feature is "unfair" is a bad reason. That's like saying lizards having a weakness to cold is unfair.

**Comment by VaelophisNyx** [comment]
Thing is it should never have been opt in. This is a good change.

**Comment by N3D6** [comment]
Wow this pr fucking blows, forced features about a system that sucks and is extremely easy to exploit is dumb, let it be an opt in. It's not "unfair" to people who have it on, if they don't wanna have a disadvantage, turn it the fuck off

**Comment by JamieD1** [comment]
I agree it should be on for everyone and this should be merged, however if I did people would burn my house down because it doesn't fit our playerbase. Sorry Moogle.

**Pull request closed without merging**